### DISCUSSION TITLE
Help with Editing .exe files!!!

### DISCUSSION THREAD

**Discussion by mediocrecartoonist** [NationalSecurityAgency/ghidra#3691] [Q&A]
I'm trying to edit a crackme.exe file to get rid of the malicious content inside of it, but I can't figure out how to edit the code and keep it edited. It keeps changing back to the original code. I'm pretty new at Ghidra, so could anybody help me with this?

**Comment by ryanmkurtz** [comment]
Right-click on an instruction, and then click `Patch Instruction`.  When you are ready to save out your new file, go to **File -> Export Program** and choose **PE** from the Format list.  This is the extent of what you can currently do in terms of modifying a program and writing out legitimate binaries that will run.
